catroid/src/main/java/org/catrobat/catroid/visualplacement/VisualPlacementActivity.java

```diff
@@ -461,4 +463,27 @@ public void setYCoordinate(float yCoordinate) {
 			yCoord = yCoordinate;
 		}
 	}
+
+	@VisibleForTesting(otherwise = MODE_PRIVATE)
+	public void drawAxes() {
+		Canvas canvas = new Canvas(((BitmapDrawable) frameLayout.getBackground()).getBitmap());
+		Paint paint = new Paint();
+		float layoutWidthHalf = layoutWidth / 2f;
+		float layoutHeightHalf = layoutHeight / 2f;
+		paint.setColor(Color.RED);
```

Comment:
I think it is a bad idea to just use a fixed color for this. - What would happen if someone chose a red background?

--> If you want to just use a line, you'd need to pick some kind of automatic contrast color to the background. (E.g. use a 16x16 grid, calculate the average color and use the contrast color for the pixel in the middle.)

Alternatively, you could just average over the whole image and pick the "farthest away" in a list of 3-4 possible colors. - This would probably be the easiest, while still yielding good results.